### ISSUE TITLE
Undefined is not a function since 7.1.2

### ISSUE THREAD

**Issue by dhoko** [postcss/postcss-import#92]
Hello !

With the latest version, the compilation doesn't work anymore :cry: (*it was ok with the 7.1.1*)

```
[11:55:06] Using gulpfile ~/dev/front-app/src/GulpFile.js
[11:55:06] Starting 'styles'...
[11:55:06] Finished 'styles' after 21 ms
[11:55:06] Plumber found unhandled error:
 TypeError in plugin 'gulp-postcss'
Message:
   /src/styles/index.css:1:1: undefined is not a function
Details:
    originalMessage: undefined is not a function
    fileName: /src/styles/index.css
    lineNumber: 1
    columnNumber: 1
```

index.css
```css
@import 'reset.css';
@import 'common.css';
@import 'buttons.css';
```

```sh
$ node -v 
v0.12.7
```

**Comment by TrySound** [comment]
@dhoko Thanks for catch. Will be fixed asap

**Comment by MoOx** [comment]
Issue is related to node 0.12.
Maybe it's time to update since node 5 is out :p

**Comment by MoOx** [comment]
Closed by 7ebe6550efa6736e9b965d516a84ffcd399a6565

**Comment by dhoko** [comment]
![](http://i.memeful.com/media/post/Wdnj5Vd_700wa_0.gif)
